hooks/live-samples.js

```diff
@@ -12,12 +12,14 @@ for (const iframe of document.querySelectorAll("iframe[data-live-id]")) {
       /** @type {MDNCodeExample[]} */
       const codeExamples = [];
 
-      for (const element of [
+      // Using Set, as some code examples have both classes.
+      const elements = new Set([
         // use getElementsByClassName as liveId can contain dots,
         // which breaks querySelectorAll
         ...document.getElementsByClassName(`live-sample___${liveId}`),
         ...document.getElementsByClassName(`live-sample---${liveId}`),
-      ]) {
+      ]);
+      for (const element of elements.values()) {
```

Comment:
@LeoMcA Maybe a better alternative would be to go back to `querySelectorAll()`, but escape the `liveId`?

Reply:
Yes, I was just thinking the same

Reply:
cf714edad7be8c173341223edf461f23be36c2a0